### ISSUE TITLE
Some little adjustments for Wood Theme

### ISSUE THREAD

**Issue by alemercu** [DS-Homebrew/TWiLightMenu#2427]
**Is your feature request related to a problem? Please describe.**
There are some differences between original Wood UI and Wood theme of TWiLight Menu:
1. Highlight animation of icons
2. Selected game while scrolling
3. Text's font

**Describe the solution you'd like**
I made some thests because I have a DS lite (white) with wood ui and a DSi (blue) with twilight menu so i can see clearly the differences between the two.

1. The highlight animation of the icons is too fast, i take a video of the screen and slowed to match the speed of the original Wood UI.
The correct speed is x0.57 of the actual speed, more or less.

(white DS is always original wood ui and the blue one is the wood theme)

x1 Wood UI:

https://github.com/DS-Homebrew/TWiLightMenu/assets/83449530/b210db40-ba5f-404e-9284-84ebc0a8862a

x1 Wood theme:

https://github.com/DS-Homebrew/TWiLightMenu/assets/83449530/8a4ee631-3b5c-4306-821f-f21c95670402

x0.57 Wood theme:

https://github.com/DS-Homebrew/TWiLightMenu/assets/83449530/27a638ce-f70f-4d9c-bc66-62d2c4a6c6c5

2. Selected game while scrolling. In original Wood UI while scrolling down the selected game is alway the last one on the bottom of the screen, and scrolling up is always the first.
In Wood theme instead are always used the two game in the middle of the creed, a little bit weird.

https://github.com/DS-Homebrew/TWiLightMenu/assets/83449530/e2db2b66-d9b9-4ebb-bba7-6f5950ffc62c

3. Text's font. It is not a real problem, I only report this to inform about this. Maybe the lines of text of the title are a little too close to each other, beacause the font is bigger than the original font.

![collage](https://github.com/DS-Homebrew/TWiLightMenu/assets/83449530/467c80bc-f2b2-4b36-ba6d-03a9a20615fd)

**Describe alternatives you've considered**
None

**Additional context**
TWiLight Menu v27.4.0 and Wood UI v1.62

Thank you very very much for all the work and support made in this years. You are incredible <3.

**Comment by RocketRobz** [comment]
> 2. Selected game while scrolling. In original Wood UI while scrolling down the selected game is alway the last one on the bottom of the screen, and scrolling up is always the first.
>    In Wood theme instead are always used the two game in the middle of the creed, a little bit weird.

When Wood UI theme got re-implemented, the scrolling was exactly like the original, but later on, someone made a PR which improved upon scrolling, and so now the selection bar only reaches the bottom when the end of the ROM list is reached.
As a result, you can now scroll by holding the top or bottom selection to scroll through the list via touch.

**Comment by stripes-8** [comment]
Yep, what Robz said about the scrolling thing is true, and I'm the one who's at fault for that.
I thought it would be a good idea because it would make it easier to know if the selection bar had reached the end of the list. I didn't think it would be a problem at first, but maybe it would be better if this feature is optional?
As for the font, I actually did start making a new font similar to the original a while, but it doesn't cover every character for every language, so I eventually stopped and just used it for personal use. I could submit what I started if anyone else wants to work on it too.
